eo-runtime/src/main/java/org/eolang/SafeFunc.java

```diff
@@ -47,18 +47,17 @@ class SafeFunc<T> implements Supplier<T> {
     }
 
     @Override
+    @SuppressWarnings({"PMD.AvoidCatchingGenericException", "PMD.AvoidRethrowingException"})
     public T get() {
         try {
             return this.origin.call();
         } catch (final InterruptedException ex) {
             Thread.currentThread().interrupt();
-            throw new ExInterrupted();
-        } catch (final ExAbstract ex) {
```

Comment:
@c71n93 there was a reason why I added this `ExAbstract` catch block but I forgot to document it. There are a lot of places in the code where we throw some exceptions that extends `ExAbstract`. But today `ExAbstract` extends `RuntimeException`, tomorrow it may be just `Exception`. So the main purpose of this block is to explicitly show that this method catches `ExAbstract` exception which is important for such objects like `PhSafe` or `AtSafe`